src/js/site/main.js

```diff
@@ -395,3 +395,17 @@ $(document).ready(function () {
       }
     });
 });
+
+// insights page column highlights
+// highlight will only work if inside of #inights-table
+$(document).ready(function () {
+  var urlParts = window.location.href.split('#');
+  if (urlParts && urlParts[1]) {
+    var highlight = urlParts[1].split('?');
+    if (highlight[1] == 'highlight') {
+      $('#' + highlight[0]).each(function () {
+        $(this).attr('class', 'highlight');
+      });
+    }
+  }
```

Comment:
I decided to use jquery and css to do the highlight instead of using a styled <mark></mark>

Reply:
Let's add some tests for this please 🙏